client/src/Components/com-list-species.js

```diff
@@ -1,17 +1,17 @@
 import React from "react";
 
-const listRender = (list) => {
```

Comment:
this is better due we are not defining function on each re-render 
and learn they can separate list renders and other etc. functions